src/App.tsx

```diff
@@ -1,39 +1,65 @@
-import React from 'react';
+/* eslint-disable no-console */
+import { useState, useEffect } from 'react';
 import './App.scss';
 
 function getRandomName(): string {
-  const value = Date.now().toString().slice(-4);
-
-  return `Clock-${value}`;
+  return `Clock-${Date.now().toString().slice(-4)}`;
 }
 
 export const App: React.FC = () => {
-  const today = new Date();
-  let clockName = 'Clock-0';
+  const [hasClock, setHasClock] = useState(true);
+  const [clockName, setClockName] = useState('Clock-0');
+  const [time, setTime] = useState(new Date());
 
-  // This code starts a timer
-  const timerId = window.setInterval(() => {
-    clockName = getRandomName();
-  }, 3300);
+  useEffect(() => {
+    const leftClick = () => {
+      setHasClock(true);
+    };
 
-  // this code stops the timer
-  window.clearInterval(timerId);
+    const rightClick = (event: MouseEvent) => {
+      event.preventDefault();
+      setHasClock(false);
+    };
 
-  return (
-    <div className="App">
-      <h1>React clock</h1>
+    const timerID = window.setInterval(() => {
+      setClockName(getRandomName());
+    }, 3300);
+
+    const clockTimerID = window.setInterval(() => {
+      setTime(new Date());
+    }, 1000);
 
-      <div className="Clock">
-        <strong className="Clock__name">
-          {clockName}
-        </strong>
+    document.addEventListener('click', leftClick);
+    document.addEventListener('contextmenu', rightClick);
 
-        {' time is '}
+    return () => {
+      document.removeEventListener('click', leftClick);
+      document.removeEventListener('contextmenu', rightClick);
+      window.clearInterval(timerID);
+      window.clearInterval(clockTimerID);
+    };
+  }, []);
 
-        <span className="Clock__time">
-          {today.toUTCString().slice(-12, -4)}
-        </span>
-      </div>
+  useEffect(() => {
+    console.log(time);
```

Comment:
You should use `console.info` method, not the `console.log`:
![image](https://user-images.githubusercontent.com/25587299/227573468-94153b5e-6251-4f42-acfe-3a5443826451.png)

Reply:
Maybe it's something wrong with the test. Never mind.